src/index.html

```diff
@@ -22,6 +22,321 @@
+            <p class="card__product-code">Product code: 195434</p>
+            <div class="card__product-rating">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price-box">
+              <p class="card__price-label">Price:</p>
+              <span class="card__price-value">$2,199</span>
+            </div>
+          </div>
+          <button
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </button>
+        </div>
+        <div class="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="Product"
+          />
+          <div class="card__content">
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+            <p class="card__product-code">Product code: 195434</p>
+            <div class="card__product-rating">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price-box">
+              <p class="card__price-label">Price:</p>
+              <span class="card__price-value">$2,199</span>
+            </div>
+          </div>
+          <button class="card__button">Buy</button>
+        </div>
+        <div class="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="Product"
+          />
+          <div class="card__content">
+            <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h1>
+            <p class="card__product-code">Product code: 195434</p>
+            <div class="card__product-rating">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__price-box">
+              <p class="card__price-label">Price:</p>
+              <span class="card__price-value">$2,199</span>
+            </div>
+          </div>
+          <button class="card__button">Buy</button>
```

Comment:
The `data-qa` attribute is missing for this button. Ensure consistency by adding `data-qa` attributes to all similar elements if needed for testing or automation.